CONTRIBUTING.md

```diff
@@ -36,20 +36,31 @@ Demos and examples 🤓 can be found on [fabricjs.com](http://fabricjs.com/demos
 
 **These are minimal requirements. Without them issues shall be ⛔.**
 
-If it's not a bug **OR** if you're unsure, start a 🤠 [discussion](https://github.com/fabricjs/fabric.js/discussions) or create a post ✉️ on [Fabric's google group](groups.google.com/forum/?fromgroups#!forum/fabricjs) where someone might clarify some of the things.
+If it's not a bug **OR** if you're unsure, start a 🤠 [discussion](https://github.com/fabricjs/fabric.js/discussions) or create a post ✉️ on [Fabric's google group](https://groups.google.com/g/fabricjs) where someone might clarify some of the things.
+
+Check out [**Helping Out**](#%EF%B8%8F-helping-out).
 
 ## 🚧🎢 Developing 💡✨
 
 ### Getting Started
 
+#### Setting up locally
+
 1. 🍴 Fork the repository
 1. 💾 Clone your 🍴 to your 💻
 1. Install dependencies 🕹️ `npm i --include=dev`
 1. Next Up [Prototyping](#-prototyping) & [Testing](#-testing)
 
+#### Online
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/from-referrer/)
+
+Gitpod will start the [prototyping](#-prototyping) apps and expose them as endpoints.
+`A service is available on port ...` popups will show up.
+
+
 ### 🧭 Prototyping
```

Comment:
both those sections i would move them down.
I would like contributing to be ordered in difficulty ramping up.
- fix typos
- contribute to docs
- report and fix a bug with a test
- do PRs with cool prototypes shown online
^^^ this last step is an excpetion, not the norm